docs/tools/cli/mirrormaker.rst

```diff
@@ -0,0 +1,221 @@
+  * - ``--source-cluster``
+    - The Aiven for Apache Kafka service name to be used as source for replication
+  * - ``--target-cluster``
+    - The Aiven for Apache Kafka service name to be used as target for replication
+
+
+**Example:** In the service ``kafka-mm`` retrieve the details of the replication flow from an Aiven for Apache Kafka service with integration alias ``kafka-source-alias`` to the service named ``kafka-target-alias``.
+
+::
+
+  avn mirrormaker replication-flow get kafka-mm \
+    --source-cluster kafka-source-alias \
+    --target-cluster kafka-target-alias
+
+An example of the ``avn mirrormaker replication-flow get`` command output:
+
+.. code:: json
+
+    {
+        "emit_heartbeats_enabled": true,
+        "enabled": true,
+        "replication_policy_class": "org.apache.kafka.connect.mirror.DefaultReplicationPolicy",
+        "source_cluster": "kafka-source-alias",
+        "sync_group_offsets_enabled": true,
+        "sync_group_offsets_interval_seconds": 60,
+        "target_cluster": "kafka-target-alias",
+        "topics": [
+            "my-src-topic.*"
+        ],
+        "topics.blacklist": [
+            ".*not-include"
+        ]
+    }
+
+``avn mirrormaker replication-flow list``
+'''''''''''''''''''''''''''''''''''''''''''
+
+Lists the configuration details for all replication flows defined in an existing Aiven for Apache Kafka MirrorMaker2 service. 
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The Aiven for Apache Kafka MirrorMaker2 service name where to create the replication flow
+
+
+**Example:** List the configuration details for all replication flows defined in an existing Aiven for Apache Kafka MirrorMaker2 named ``kafka-mm``.
+
+::
+
+  avn mirrormaker replication-flow list kafka-mm 
+
+An example of the ``avn mirrormaker replication-flow list`` command output:
+
+.. code:: json
+
+    [
+          {
+              "emit_heartbeats_enabled": true,
+              "enabled": true,
+              "replication_policy_class": "org.apache.kafka.connect.mirror.DefaultReplicationPolicy",
+              "source_cluster": "kafka-source-alias",
+              "sync_group_offsets_enabled": true,
+              "sync_group_offsets_interval_seconds": 60,
+              "target_cluster": "kafka-target-alias",
+              "topics": [
+                  "my-src-topic.*"
+              ],
+              "topics.blacklist": [
+                  ".*not-include"
+              ]
+          }
+     ]
+
+``avn mirrormaker replication-flow update``
+'''''''''''''''''''''''''''''''''''''''''''
+
+Updates an existing Aiven for Apache Kafka MirrorMaker2 replication flow. 
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The Aiven for Apache Kafka MirrorMaker2 service name where to create the replication flow
```

Comment:
same as above - not creating, but updating

Reply:
done thanks!